admin/class-recalibration-beta.php

```diff
@@ -127,6 +127,17 @@ public static function is_enabled() {
 		return WPSEO_Options::get( 'recalibration_beta' );
 	}
 
+	/**
+	 * Checks if the recalibration beta should use the local file.
+	 *
+	 * If false, the my-yoast-proxy should be used.
+	 *
+	 * @return bool Whether the local file should be used.
+	 */
+	public static function use_local_file() {
```

Comment:
The only usage for this is in the `admin/class-admin-asset-analysis-worker-location.php`. I suggest to move it to that file and makes the method non-static and protected. 

The reason for this is that you are introducing a public method that is only meant for the recalibration, when that beta is over we want to easily remove the code that belongs to this beta.

Reply:
Changed!